lib/classes/PluginManager.js

```diff
@@ -641,6 +642,10 @@ class PluginManager {
       }
     });
   }
+
+  asyncPluginInit() {
+    return BbPromise.map(this.plugins, plugin => plugin.asyncInit && plugin.asyncInit());
```

Comment:
(sorry, didn't look carefully on this earlier)

We should avoid using non standard Promise API's (to ease future migration), I think it's safe to use here `BbPromise.all`

Reply:
Just pushed a commit which addresses this...